### ISSUE TITLE
Error running from source

### ISSUE THREAD

**Issue by apandada1** [rafaelmardojai/WebfontKitGenerator#12]
OS: Linux Mint 20.1
Python: 3.8.5

I installed the latest master branch (commit https://github.com/rafaelmardojai/WebfontKitGenerator/commit/5d6c7dcfbdbecc7f9af03c683e444a4863b46d2a ) of WebfontKitGenerator from source, and got this error when I tried to run it (the daily build PPA shows the same error too).
```
$ webfontkitgenerator

(webfontkitgenerator:17526): Gtk-CRITICAL **: 11:27:27.480: Error building template class 'Window' for an instance of type 'Window': .:101:1 Invalid object type 'HdyStatusPage'
Traceback (most recent call last):
  File "/home/archisman/.local/share/webfontkitgenerator/webfontkitgenerator/main.py", line 45, in do_activate
    self.window = Window(application=self)
  File "/home/archisman/.local/share/webfontkitgenerator/webfontkitgenerator/window.py", line 52, in __init__
    self.setup_widgets()
  File "/home/archisman/.local/share/webfontkitgenerator/webfontkitgenerator/window.py", line 63, in setup_widgets
    self.stack.add_titled(self.options, 'options', _('Options'))
AttributeError: 'NoneType' object has no attribute 'add_titled'
```

**Comment by apandada1** [comment]
Is it because I have libhandy 1.0 (not 1.1) in my system?

**Comment by rafaelmardojai** [comment]
> Is it because I have libhandy 1.0 (not 1.1) in my system?

Yes, in `master` I have updated to  libhandy 1.1 to use `HdyStatusPage`.

**Comment by apandada1** [comment]
Even Arch Linux does not have libhandy 1.1 yet in their repo :-)

**Comment by rafaelmardojai** [comment]
Good to know, I will wait it to propagate before releasing a new version.

**Comment by apandada1** [comment]
Now libhandy 1.2 is available in Arch, and libhandy 1.1.90 in Ubuntu 21.04 (current development version).
